docs/_snippets/vitest-plugin-vitest-workspace.md

```diff
@@ -20,6 +19,7 @@ export default defineWorkspace([
       // storybookNextjsPlugin(),
     ],
     test: {
+      name: 'storybook',
```

Comment:
@pedrosousa13 can you adjust the other examples in this file as well, not only for React, so that we're able to provide the information accurately for all the supported frameworks?